### ISSUE TITLE
Terminal shortcuts

### ISSUE THREAD

**Issue by ovidius72** [zed-industries/zed#14472]
### Check for existing issues

- [X] Completed

### Describe the feature

Please provide a way to open the Teminal in a zoomed in panel or, if doable, in a floating panel/window/dialog with a customizable shortcut.
Thanks.

**Comment by Moshyfawn** [comment]
Do you want something like `"shift-escape": "workspace::ToggleZoom"` when the terminal panel is focused?

**Comment by ovidius72** [comment]
> Do you want something like `"shift-escape": "workspace::ToggleZoom"` when the terminal panel is focused?

No, that's already supported. I'd like to have a shortcut that open up the terminal already zoomed.  No matter if i'm in a pane, project or terminal panel.

**Comment by notpeter** [comment]
Potentially a duplicate/similar to:
- https://github.com/zed-industries/zed/issues/14472

**Comment by niluke** [comment]
terminal opens with `ctrl` + ```  `  ``` key but it doesn't  close with  same key  bindings

**Comment by ovidius72** [comment]
> terminal opens with `ctrl` + `` ` `` key but it doesn't close with same key bindings

and, as far as i know, it does not open zoomed in. That's what i'm asking for. I want to leverage on this feature when writing tasks.
